client/src/components/CurrentWeather.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from "react";
+import { getCityCoordinates, getWeatherForCoordinates } from "./CurrentWeatherFunctions";
```

Comment:
zase jenom zvyk, ale nazvy souboru, pokud to nejsou 
- komponenty
- tridy/interface
- testy nebo page objecty komponent

zacinaji malymi pismeny